examples/zkp/xor_compat/verify.go

```diff
@@ -68,11 +68,11 @@ func VerifyProof(a []byte, b []byte, c []byte, publicInput [][]byte) bool {
 		panic("error: inputlen or iclen")
 	}
 	icPoints := make([]crypto.Bls12381Point, iclen)
-	for i := 0; i < iclen; i++ {
+	for i := range iclen {
```

Comment:
This breaks the contract, our compiler can't handle it.

Reply:
#3525

Reply:
Added a comment.

Reply:
You can range over `icPoints` here, that'd work.

Reply:
Fixed.